src/components/archive.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Icon from "./icon";
+import Link from "./link";
+
+const environment = process.env.GATSBY_ENVIRONMENT_BRANCH;
+
+export default function Archive({ title, path, ...props }) {
+  const url = `https://github.com/EnterpriseDB/docs-archive/raw/${environment}${path}`;
```

Comment:
I would consider adding a leading slash to `path` if the user doesn't supply one.

Reply:
👍